biz.aQute.bnd.annotation/src/aQute/bnd/annotation/Constants.java

```diff
@@ -0,0 +1,25 @@
+package aQute.bnd.annotation;
+
+public interface Constants {
```

Comment:
I prefer we use a class (not an interface) for constant definition. See Effective Java, 2nd Ed `#19`.